### PR TITLE
Add toggleable theme button and ability to swap between compacted and expanded PersonCards

### LINKED ISSUE #118: Buttons to change theme should be a toggle instead
It does not make sense to be able to click on the light theme button while already in light theme mode. Instead maybe a single button that toggles between light and dark theme would be better.

### LINKED ISSUE #119: Allow user to swap between compacted and expanded cards
Previously everything was expanded, now everything is compacted. There are use cases for both compacted and expanded. Users should be able to toggle between these two modes. Now if they wish to swap between the two they can instead of having to click on every card every time to expand everything every time.

### PR DESCRIPTION
**Pull request by KeithPJX**
Fixes #118 and fixes #119.

Added 2 new buttons under the Appearance tab and removed Theme tab.
1. Button that toggles between light theme and dark theme
2. Button that toggles between compacted and expanded PersonCards. If in compact mode, all cards are by default in their compact mode and vice versa.

image

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -171,34 +189,42 @@ private void handleExit() {
      * Changes theme to light theme
      */
```
Comment (ryanlml) [inline src/main/java/seedu/address/ui/MainWindow.java]: Don't forget to update the Javadoc. Perhaps `toggles between light and dark theme` fits this better.

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -171,34 +189,42 @@ private void handleExit() {
...
-    private void handleDarkTheme() {
-        boolean hasLightTheme = primaryStage.getScene().getStylesheets().contains(lightTheme);
-        boolean hasDarkTheme = primaryStage.getScene().getStylesheets().contains(darkTheme);
-
-        if (!hasDarkTheme && hasLightTheme) {
-            primaryStage.getScene().getStylesheets().remove(lightTheme);
-            primaryStage.getScene().getStylesheets().add(darkTheme);
-            primaryStage.getScene().getStylesheets().remove(extensionsLight);
-            primaryStage.getScene().getStylesheets().add(extensionsDark);
+    private void handleCompactExpand() {
```
Comment (ryanlml) [inline src/main/java/seedu/address/ui/MainWindow.java:219]: May want to consider adding Javadocs for `handleCompactExpand` too.

**Review by ryanlml — APPROVED** [review]
LGTM! Javadocs can be updated. Thanks for the changes!

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (KeithPJX) [reply src/main/java/seedu/address/ui/MainWindow.java]: Oh yea, thanks for the catch.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (KeithPJX) [reply src/main/java/seedu/address/ui/MainWindow.java:219]: Yea will do.

**Pull request merged**